dip-0006.md

```diff
@@ -364,12 +364,13 @@ The internal Dash message name is `qfcommit` and the format of the message is:
 
 #### Current LLMQ types
 
-| Type | Name | Total Members | Threshold Members | Duration |
-|--|--|--|--|--|
-| 1 | LLMQ_50_60 | 50 | 30 (60%) | 1 Hour |
-| 2 | LLMQ_400_60 | 400 | 240 (60%) | 12 Hours |
-| 3 | LLMQ_400_85 | 400 | 340 (85%) | 24 Hours |
-| 256 | LLMQ_NONE (Reserved) | - | - | - |
+| Type | Name | Total Members | Threshold Members | Duration | Notes |
+|--|--|--|--|--|--|
+| 1 | LLMQ_50_60 | 50 | 30 (60%) | 1 Hour | |
+| 2 | LLMQ_400_60 | 400 | 240 (60%) | 12 Hours | |
+| 3 | LLMQ_400_85 | 400 | 340 (85%) | 24 Hours | |
+| 100 | LLMQ_10_60 | 10 | 6 (60%) | 1 Hour | For testing only |
+| 256 | LLMQ_NONE | - | - | - | Reserved |
```

Comment:
It's actually not "Reserved" but meant as a place holder when the type is not initialized.

Reply:
Should the description be changed to something like "Uninitialized"? Or remove the line entirely?

Reply:
Probably remove the line completely as it's internal-only and not part of consensus.

Reply:
Okay, updated :+1: